### PR TITLE
Update documentation discrepancies

### PR DESCRIPTION
**Pull request by ptrelease**
Corrected documentation where we discovered upon migration that some proposed tables didn't pan out.

### CODE REVIEW

File: source/documentation/data_dictionary/tables/_declarations.md [hunk 1]
```diff
@@ -12,4 +12,6 @@ searching within the JSON.
   declaration_type: String      The type of declaration
   json: JSON                    A structured version of a declaration
```
Comment (stuartmccoll) [inline source/documentation/data_dictionary/tables/_declarations.md:13]: We've got this in as a `jsonb` column.

File: source/documentation/data_dictionary/tables/_funding_applications.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ## funding_applications
 
-Table that contains details specific to a funding application.
+Table that contains details specific to a funding application.  The current application table may contain additional rows as originally declaration data was kept on this table.
```
Comment (stuartmccoll) [inline source/documentation/data_dictionary/tables/_funding_applications.md:3]: What do we mean by 'additional rows' here?

File: source/documentation/data_dictionary/tables/_organisations_org_types.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+## organisation_org_types
+
+Link table between and organisation and organisation types.
```
Comment (stuartmccoll) [inline source/documentation/data_dictionary/tables/_organisations_org_types.md:3]:
```suggestion
Link table between organisations and organisation types.
```

File: source/documentation/data_dictionary/tables/_organisations_org_types.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+## organisation_org_types
```
Comment (stuartmccoll) [inline source/documentation/data_dictionary/tables/_organisations_org_types.md:1]:
```suggestion
## organisations_org_types
```

**Review by stuartmccoll — CHANGES_REQUESTED** [review]

File: source/documentation/data_dictionary/tables/_cash_contributions.md [hunk 5]
```diff
@@ -1,13 +1,13 @@
...
-secured_type_id: Integer            link to more data on the secured_type table
-created_at: Timestamp
-updated_at: Timestamp
+## cash contributions
+
+details about cash contributions
+
+```
+id: UUID <<PK>>
+application_id: UUID <<FK>>         Currently called project Id.
```
[the remark this answers is not in the document]
Comment (ptrelease) [reply source/documentation/data_dictionary/tables/_cash_contributions.md:7]: changed

File: source/documentation/data_dictionary/tables/_declarations.md [hunk 6]
```diff
@@ -12,4 +12,6 @@ searching within the JSON.
...
   version: Integer              The version of the declaration
+  created_at : Timestamp
+  updated_at : Timestamp
```
[the remark this answers is not in the document]
Comment (ptrelease) [reply source/documentation/data_dictionary/tables/_declarations.md:16]: changed

File: source/documentation/data_dictionary/tables/_declarations.md (hunk 1 above)
Comment (ptrelease) [reply source/documentation/data_dictionary/tables/_declarations.md:13]: changed

File: source/documentation/data_dictionary/tables/_funding_applications.md (hunk 2 above)
Comment (ptrelease) [reply source/documentation/data_dictionary/tables/_funding_applications.md:3]: Changed to fields

**Review by stuartmccoll — APPROVED** [review]

**Pull request merged**